src/communication/network_interface_cl.h

```diff
@@ -119,7 +119,7 @@ extern int heap_create (HFID * hfid, const OID * class_oid, bool reuse_oid);
 #if defined(ENABLE_UNUSED_FUNCTION)
 extern int heap_destroy (const HFID * hfid);
 #endif
-extern int heap_destroy_newly_created (const HFID * hfid, const OID * class_oid);
+extern int heap_destroy_newly_created (const HFID * hfid, const OID * class_oid, const int force = 0);
```

Comment:
how about using bool type instead of int?

Reply:
At first, I did as you said. However, this is a network interface function, and that interfaces use int for a boolean type. It seems because there is no `or_pack_bool()` function. It is for code consistency.

Reply:
Oh, I've found out `heap_create` is the same sort of function, but it uses boolean as a parameter. It turns out case by case. I'll follow your advice.